### PR TITLE
[M1612] Change bleaching percent data type to decimal

### PR DESCRIPTION
**Pull request by gridcell**
- changes to models and ingest

### CODE REVIEW

**Comment by gridcell** [comment]
[bleaching perc_cover columns - add support for numeric entries (currently only integer)](https://trello.com/c/KWvxwoEY/1612-bleaching-perccover-columns-add-support-for-numeric-entries-currently-only-integer)

File: src/api/ingest/bleaching_serializer.py [hunk 1]
```diff
@@ -65,6 +65,7 @@ def group_records(self, records):
         for rec in grouped_records:
             for n, obs in enumerate(rec["data"].get("obs_quadrat_benthic_percent")):
                 obs["quadrat_number"] = n + 1
+        print(f"grouped_records: {grouped_records}")
```
Comment (saanobhaai) [inline src/api/ingest/bleaching_serializer.py:68]: remove

File: src/api/tests/test_sample_unit_method_obs.py [hunk 2]
```diff
@@ -344,6 +345,6 @@ def test_bleaching_quadrat_benthic_percent_field_report(
     assert float(rows[3]["Latitude"]) == site1.location.y
     assert float(rows[3]["Longitude"]) == site1.location.x
     assert rows[3]["Observers"] == profile1.full_name
-    assert rows[3]["Soft coral (% cover)"] == str(
-        obs_quadrat_benthic_percent1_4.percent_soft
+    assert pytest.approx(float(rows[3]["Soft coral (% cover)"]), 1) == pytest.approx(
```
Comment (saanobhaai) [inline src/api/tests/test_sample_unit_method_obs.py:348]: Interesting. I'm sure it's right, but curious why `float` and not `Decimal` (and wouldn't that remove the need for the `approx`).

**Review by saanobhaai — CHANGES_REQUESTED** [review]
Just a print statement

**Pull request merged**